src/dmd/backend/cod3.d

```diff
@@ -2066,39 +2070,39 @@ else
 {
         i = (config.inline8087) ? (3 + cmporder87(e.EV.E2)) : 2;
 }
-  }
-  else if (tyuns(tym) || tyuns(e.EV.E2.Ety))
+    }
+    else if (tyuns(tym) || tyuns(e.EV.E2.Ety))
         i = 1;
-  else if (tyintegral(tym) || typtr(tym))
+    else if (tyintegral(tym) || typtr(tym))
         i = 0;
-  else
-  {
+    else
+    {
         debug
         elem_print(e);
```

Comment:
Indent.

Reply:
I've been trying different styles for that, not sure which is best.